src/server/Document/component.jsx

```diff
@@ -100,6 +100,7 @@ const Document = ({
       <body {...ampGeoPendingAttrs}>
         <div id="root" dangerouslySetInnerHTML={{ __html: html }} />
         {scriptsAllowed && (
+          // This script should be the first script tag in the body, otherwise Opera Mini has trouble parsing the `window.SIMORGH_DATA` object
```

Comment:
Is there a way we could add a unit test for this, to prevent it from happening in future?

Reply:
Yea good call, added.